MANIFEST.in

```diff
@@ -2,3 +2,4 @@ include versioneer.py
 include skbeam/_version.py
 include requirements.txt
 include optional-requirements.txt
+include src/ctrans.h
```

Comment:
Maybe we should do `src/*.h` for future-proofing? And don't we want the `*.c` file(s) as well?  In reference to...

> In the process, we have removed wildcards and made declarations more explicit. If we are to add any extra cython code, we must declare it in the setup.py file.

I think it makes sense to be explicit in the `setup.py` but I don't see any reason to avoid wildhards in `MANIFEST.in`.

Reply:
I think we may need to include `*.c` files as well.

Reply:
@danielballan, I tested installation (`sdist`) from the master branch, and the `*.c` files are in the resulted `.tar.gz` file.